### PR TITLE
Update fit2obs for COM refactor

### LINKED ISSUE #1486: Update Fit2Obs for COM refactor
**Description**
The Fit2Obs job and external package were not updated as part of the initial COM refactor work. Work will need to be done to update the workflow scripts, as well as update the external Fit2Obs package, to utilize the new structure and templates.

**Requirements**
- A Fit2Obs job that uses the COM templates to generate COM directory paths.
- A Fit2Obs package that expects full paths to COM directories (including dates) as inputs, and changes to modify those paths for other cycles if needed.

**Acceptance Criteria (Definition of Done)**
Able to run Fit2Obs successfully.

Dependency: #1421

### PR DESCRIPTION
**Pull request by WalterKolczynski-NOAA**
**Description**

Updates the fit2obs j-job for COM refactor. Also corrects a related error in the archive job where the wrong source was being used for history files to be copied to the verification archive.

Fixes #1486

**Type of change**

- [x] Bug fix (non-breaking change which fixes an issue)

**How Has This Been Tested?**

- [x] Cycled test on Orion
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes

### CODE REVIEW

**Comment by WalterKolczynski-NOAA** [comment]
@jack-woollen I get the following error in the logs, though I think it is unrelated to the path change:

```
+ excfs_gdas_vrfyfits.sh[187]: cp -p /work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.37929/fitx/adpupa.mand.fcs.2021122200 /work/noaa/global/wkolczyn/archive/fit2obs/horiz/fcs/adpupa.mand.2021122200
cp: cannot stat '/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.37929/fitx/adpupa.mand.fcs.2021122200': No such file or directory
+ excfs_gdas_vrfyfits.sh[188]: cp -p /work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.37929/fitx/adpsfc.fcs.2021122200 /work/noaa/global/wkolczyn/archive/fit2obs/horiz/fcs/adpsfc.2021122200
cp: cannot stat '/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.37929/fitx/adpsfc.fcs.2021122200': No such file or directory
+ excfs_gdas_vrfyfits.sh[189]: cp -p /work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.37929/fitx/sfcshp.fcs.2021122200 /work/noaa/global/wkolczyn/archive/fit2obs/horiz/fcs/sfcshp.2021122200
cp: cannot stat '/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.37929/fitx/sfcshp.fcs.2021122200': No such file or directory
+ excfs_gdas_vrfyfits.sh[190]: cp -p /work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.37929/fitx/aircar.fcs.2021122200 /work/noaa/global/wkolczyn/archive/fit2obs/horiz/fcs/aircar.2021122200
cp: cannot stat '/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.37929/fitx/aircar.fcs.2021122200': No such file or directory
+ excfs_gdas_vrfyfits.sh[191]: cp -p /work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.37929/fitx/aircft.fcs.2021122200 /work/noaa/global/wkolczyn/archive/fit2obs/horiz/fcs/aircft.2021122200
cp: cannot stat '/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.37929/fitx/aircft.fcs.2021122200': No such file or directory
```
Full log: https://gist.github.com/WalterKolczynski-NOAA/d7be426a446c3444d8364c6591dda04c

Not sure if this expected or indicates another bug.

Working directory (Orion):
```
/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.37929
```

**Review by jack-woollen — COMMENTED** [review]
@WalterKolczynski-NOAA I'm home after being in the hospital for a couple days. I can't look into this until next week. Please rerun your test with the current tag. Thanks.

**Comment by jack-woollen** [comment]
@WalterKolczynski-NOAA  Okay, taking another look at the logs while more awake,  the problem is if you want to test the code and avoid missing file messages you need to provide 126 hour forecasts in the vrfyarch for 5 days prior to the verification date.

**Comment by WalterKolczynski-NOAA** [comment]
Thanks @jack-woollen, I'll run it out longer and report back. Well wishes for your recovery! This can wait until you are better.

**Comment by WalterKolczynski-NOAA** [comment]
@jack-woollen Ran out to six days and still the same files missing.
```
+ excfs_gdas_vrfyfits.sh[187]: cp -p /work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.235040/fitx/adpupa.mand.fcs.2021122600 /work/noaa/global/wkolczyn/archive/fit2obs/horiz/fcs/adpupa.mand.2021122600
cp: cannot stat â/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.235040/fitx/adpupa.mand.fcs.2021122600â: No such file or directory
+ excfs_gdas_vrfyfits.sh[188]: cp -p /work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.235040/fitx/adpsfc.fcs.2021122600 /work/noaa/global/wkolczyn/archive/fit2obs/horiz/fcs/adpsfc.2021122600
cp: cannot stat â/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.235040/fitx/adpsfc.fcs.2021122600â: No such file or directory
+ excfs_gdas_vrfyfits.sh[189]: cp -p /work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.235040/fitx/sfcshp.fcs.2021122600 /work/noaa/global/wkolczyn/archive/fit2obs/horiz/fcs/sfcshp.2021122600
cp: cannot stat â/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.235040/fitx/sfcshp.fcs.2021122600â: No such file or directory
+ excfs_gdas_vrfyfits.sh[190]: cp -p /work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.235040/fitx/aircar.fcs.2021122600 /work/noaa/global/wkolczyn/archive/fit2obs/horiz/fcs/aircar.2021122600
cp: cannot stat â/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.235040/fitx/aircar.fcs.2021122600â: No such file or directory
+ excfs_gdas_vrfyfits.sh[191]: cp -p /work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.235040/fitx/aircft.fcs.2021122600 /work/noaa/global/wkolczyn/archive/fit2obs/horiz/fcs/aircft.2021122600
cp: cannot stat â/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs/fit2obs.235040/fitx/aircft.fcs.2021122600â: No such file or directory
```

Analysis ones populate fine, just the forecast ones aren't created.

**Comment by jack-woollen** [comment]
@WalterKolczynski-NOAA For some reason still not getting the prior forecast days in the mix. Below is a list of outputs from the orion fitx directory, followed by outputs from the operational version for the other day. 

files created from the test run:

jwoollen@Orion-login-2:/work2/noaa/stmp/wkolczyn/RUNDIRS/fit2obs> ls -ltr fit2obs.235040/fitx

f00.acar.2021122600
f00.acft.2021122600
f00.raob.2021122600
f00.sfc.2021122600
f00.surf.2021122600
f06.acar.2021122600
f06.acft.2021122600
f06.raob.2021122600
f06.sfc.2021122600
f06.surf.2021122600

adpsfc.anl.2021122600
adpupa.mand.anl.2021122600
aircar.anl.2021122600
aircft.anl.2021122600
sfcshp.anl.2021122600

files created in production (different date):

./fits
f00.acar.2023070100
f00.acft.2023070100
f00.raob.2023070100
f00.sfc.2023070100
f00.surf.2023070100
f06.acar.2023070100
f06.acft.2023070100
f06.raob.2023070100
f06.sfc.2023070100
f06.surf.2023070100
f108.acar.2023070100
f108.acft.2023070100
f108.raob.2023070100
f108.sfc.2023070100
f108.surf.2023070100
f120.acar.2023070100
f120.acft.2023070100
f120.raob.2023070100
f120.sfc.2023070100
f120.surf.2023070100
f12.acar.2023070100
f12.acft.2023070100
f12.raob.2023070100
f12.sfc.2023070100
f12.surf.2023070100
f24.acar.2023070100
f24.acft.2023070100
f24.raob.2023070100
f24.sfc.2023070100
f24.surf.2023070100
f36.acar.2023070100
f36.acft.2023070100
f36.raob.2023070100
f36.sfc.2023070100
f36.surf.2023070100
f48.acar.2023070100
f48.acft.2023070100
f48.raob.2023070100
f48.sfc.2023070100
f48.surf.2023070100
f60.acar.2023070100
f60.acft.2023070100
f60.raob.2023070100
f60.sfc.2023070100
f60.surf.2023070100
f72.acar.2023070100
f72.acft.2023070100
f72.raob.2023070100
f72.sfc.2023070100
f72.surf.2023070100
f84.acar.2023070100
f84.acft.2023070100
f84.raob.2023070100
f84.sfc.2023070100
f84.surf.2023070100
f96.acar.2023070100
f96.acft.2023070100
f96.raob.2023070100
f96.sfc.2023070100
f96.surf.2023070100

./horiz/anl:
adpsfc.2023070100
adpupa.mand.2023070100
aircar.2023070100
aircft.2023070100
sfcshp.2023070100

./horiz/fcs:
adpsfc.2023070100
adpupa.mand.2023070100
aircar.2023070100
aircft.2023070100
sfcshp.2023070100

**Comment by WalterKolczynski-NOAA** [comment]
I found the previous problem (it was unrelated to fit2obs directly). I've updated the output log. Here is the file inventory now:

```
Orion-login-1 fit2obs/>ls *20211225?? */*20211225?? tracker.2021122500/gfs
atcfunix.gdas.2021122500      fits/f00.raob.2021122500  fits/f06.raob.2021122512   fits/f24.raob.2021122500  fits/f72.surf.2021122500
atcfunix.gdas.2021122506      fits/f00.raob.2021122506  fits/f06.raob.2021122518   fits/f24.sfc.2021122500   fits/f84.acar.2021122512
atcfunix.gdas.2021122512      fits/f00.raob.2021122512  fits/f06.sfc.2021122500    fits/f24.surf.2021122500  fits/f84.acft.2021122512
atcfunix.gdas.2021122518      fits/f00.raob.2021122518  fits/f06.sfc.2021122506    fits/f36.acar.2021122512  fits/f84.raob.2021122512
atcfunix.gfs.2021122500       fits/f00.sfc.2021122500   fits/f06.sfc.2021122512    fits/f36.acft.2021122512  fits/f84.sfc.2021122512
atcfunixp.gdas.2021122500     fits/f00.sfc.2021122506   fits/f06.sfc.2021122518    fits/f36.raob.2021122512  fits/f84.surf.2021122512
atcfunixp.gdas.2021122506     fits/f00.sfc.2021122512   fits/f06.surf.2021122500   fits/f36.sfc.2021122512   fits/f96.acar.2021122500
atcfunixp.gdas.2021122512     fits/f00.sfc.2021122518   fits/f06.surf.2021122506   fits/f36.surf.2021122512  fits/f96.acft.2021122500
atcfunixp.gdas.2021122518     fits/f00.surf.2021122500  fits/f06.surf.2021122512   fits/f48.acar.2021122500  fits/f96.raob.2021122500
atcfunixp.gfs.2021122500      fits/f00.surf.2021122506  fits/f06.surf.2021122518   fits/f48.acft.2021122500  fits/f96.sfc.2021122500
enkfstat.enkfgdas.2021122500  fits/f00.surf.2021122512  fits/f108.acar.2021122512  fits/f48.raob.2021122500  fits/f96.surf.2021122500
enkfstat.enkfgdas.2021122506  fits/f00.surf.2021122518  fits/f108.acft.2021122512  fits/f48.sfc.2021122500   gsistat.gdas.2021122500
enkfstat.enkfgdas.2021122512  fits/f06.acar.2021122500  fits/f108.raob.2021122512  fits/f48.surf.2021122500  gsistat.gdas.2021122506
enkfstat.enkfgdas.2021122518  fits/f06.acar.2021122506  fits/f108.sfc.2021122512   fits/f60.acar.2021122512  gsistat.gdas.2021122512
fits/f00.acar.2021122500      fits/f06.acar.2021122512  fits/f108.surf.2021122512  fits/f60.acft.2021122512  gsistat.gdas.2021122518
fits/f00.acar.2021122506      fits/f06.acar.2021122518  fits/f12.acar.2021122512   fits/f60.raob.2021122512  gsistat.gfs.2021122500
fits/f00.acar.2021122512      fits/f06.acft.2021122500  fits/f12.acft.2021122512   fits/f60.sfc.2021122512   storms.gfso.atcf_gen.2021122500
fits/f00.acar.2021122518      fits/f06.acft.2021122506  fits/f12.raob.2021122512   fits/f60.surf.2021122512  storms.gfso.atcf_gen.altg.2021122500
fits/f00.acft.2021122500      fits/f06.acft.2021122512  fits/f12.sfc.2021122512    fits/f72.acar.2021122500
fits/f00.acft.2021122506      fits/f06.acft.2021122518  fits/f12.surf.2021122512   fits/f72.acft.2021122500
fits/f00.acft.2021122512      fits/f06.raob.2021122500  fits/f24.acar.2021122500   fits/f72.raob.2021122500
fits/f00.acft.2021122518      fits/f06.raob.2021122506  fits/f24.acft.2021122500   fits/f72.sfc.2021122500

tracker.2021122500:
gfs

tracker.2021122500/gfs:
```

I don't see any obvious errors in the log. Is this complete? @jack-woollen 
Output: `/work/noaa/global/wkolczyn/archive/fit2obs`

**Comment by jack-woollen** [comment]
@WalterKolczynski-NOAA Looks better but i don't don't see any f120 files in the fits directory and there is no horiz directories. To get f120 files you need a 126 hour forecasts since it does time interpolation, as well as 5 prior days.

**Comment by WalterKolczynski-NOAA** [comment]
Sorry, missed horiz directory with that wildcard:
```
Orion-login-1 fit2obs/>ls horiz/*/*20211225??
horiz/anl/adpsfc.2021122500       horiz/anl/aircar.2021122500  horiz/anl/sfcshp.2021122500       horiz/fcs/aircar.2021122500
horiz/anl/adpsfc.2021122506       horiz/anl/aircar.2021122506  horiz/anl/sfcshp.2021122506       horiz/fcs/aircar.2021122512
horiz/anl/adpsfc.2021122512       horiz/anl/aircar.2021122512  horiz/anl/sfcshp.2021122512       horiz/fcs/aircft.2021122500
horiz/anl/adpsfc.2021122518       horiz/anl/aircar.2021122518  horiz/anl/sfcshp.2021122518       horiz/fcs/aircft.2021122512
horiz/anl/adpupa.mand.2021122500  horiz/anl/aircft.2021122500  horiz/fcs/adpsfc.2021122500       horiz/fcs/sfcshp.2021122500
horiz/anl/adpupa.mand.2021122506  horiz/anl/aircft.2021122506  horiz/fcs/adpsfc.2021122512       horiz/fcs/sfcshp.2021122512
horiz/anl/adpupa.mand.2021122512  horiz/anl/aircft.2021122512  horiz/fcs/adpupa.mand.2021122500
horiz/anl/adpupa.mand.2021122518  horiz/anl/aircft.2021122518  horiz/fcs/adpupa.mand.2021122512
```

Not sure why it cuts off at `f108` instead of `f120`. Will see if I can find anything (maybe there is an incorrect loop).

**Comment by WalterKolczynski-NOAA** [comment]
Looks like I just had to run the forecast longer:
```
Orion-login-1 fit2obs/>ls *2600 fits/*2600 horiz/*/*2600 tracker.*2600/gfs/
atcfunix.gdas.2021122600      fits/f06.acft.2021122600   fits/f24.raob.2021122600  fits/f72.sfc.2021122600           horiz/anl/aircar.2021122600
atcfunix.gfs.2021122600       fits/f06.raob.2021122600   fits/f24.sfc.2021122600   fits/f72.surf.2021122600          horiz/anl/aircft.2021122600
atcfunixp.gdas.2021122600     fits/f06.sfc.2021122600    fits/f24.surf.2021122600  fits/f96.acar.2021122600          horiz/anl/sfcshp.2021122600
atcfunixp.gfs.2021122600      fits/f06.surf.2021122600   fits/f48.acar.2021122600  fits/f96.acft.2021122600          horiz/fcs/adpsfc.2021122600
enkfstat.enkfgdas.2021122600  fits/f120.acar.2021122600  fits/f48.acft.2021122600  fits/f96.raob.2021122600          horiz/fcs/adpupa.mand.2021122600
fits/f00.acar.2021122600      fits/f120.acft.2021122600  fits/f48.raob.2021122600  fits/f96.sfc.2021122600           horiz/fcs/aircar.2021122600
fits/f00.acft.2021122600      fits/f120.raob.2021122600  fits/f48.sfc.2021122600   fits/f96.surf.2021122600          horiz/fcs/aircft.2021122600
fits/f00.raob.2021122600      fits/f120.sfc.2021122600   fits/f48.surf.2021122600  gsistat.gdas.2021122600           horiz/fcs/sfcshp.2021122600
fits/f00.sfc.2021122600       fits/f120.surf.2021122600  fits/f72.acar.2021122600  gsistat.gfs.2021122600            storms.gfso.atcf_gen.2021122600
fits/f00.surf.2021122600      fits/f24.acar.2021122600   fits/f72.acft.2021122600  horiz/anl/adpsfc.2021122600       storms.gfso.atcf_gen.altg.2021122600
fits/f06.acar.2021122600      fits/f24.acft.2021122600   fits/f72.raob.2021122600  horiz/anl/adpupa.mand.2021122600

tracker.2021122600:
gfs

tracker.2021122600/gfs/:
```

`tracker.*/gfs/` is still empty, but if it is only TC tracking, there may not be any (the period is in December). @jack-woollen

**Review by jack-woollen — APPROVED** [review]
Thanks @WalterKolczynski-NOAA, now looks good.

File: jobs/JGDAS_FIT2OBS [hunk 1]
```diff
@@ -17,11 +17,15 @@ export CDATE
 vday=${CDATE:0:8}
 vcyc=${CDATE:8:2}
 
-export COM_INA=${ROTDIR}/gdas.${vday}/${vcyc}/atmos
+YMD=${vday} HH=${vcyc} generate_com -rx COM_ATMOS_ANALYSIS_VERIF:COM_ATMOS_ANALYSIS_TMPL
```
Comment (aerorahul) [inline jobs/JGDAS_FIT2OBS]: Is `COM_ATMOS_ANALYSIS_VERIF` used in the fit2obs scripts? Or is it being used to set `COM_INA` on line 24?
Same with `COM_OBS_VERIF`

**Review by aerorahul — APPROVED** [review]
Looks good to me. 
See a question in the review below.

File: jobs/JGDAS_FIT2OBS (hunk 1 above)
Comment (WalterKolczynski-NOAA) [reply jobs/JGDAS_FIT2OBS]: Just to set `COM_INA` and `COM_PRP` below. We can cut out the middle man if you want, but I believe I followed this same pattern in other j-jobs that had to set variables for external packages.

Comment (aerorahul) [reply jobs/JGDAS_FIT2OBS]: > Just to set `COM_INA` and `COM_PRP` below. We can cut out the middle man if you want, but I believe I followed this same pattern in other j-jobs that had to set variables for external packages.

Then we should atleast eliminate `-x`.
I would rather directly set `COM_INA` and `COM_PRP` and cut out the middle man.  There is no reason to call 2 things with the same name.

**Comment by WalterKolczynski-NOAA** [comment]
> Automated global-workflow Testing Results:
> 
> ```
> Machine: Hera
> Start: Thu Jul 13 15:56:54 UTC 2023 on hfe05
> ---------------------------------------------------
> Checkout:                      *SUCCESS*
> Checkout: Completed at Thu Jul 13 16:01:41 UTC 2023
> Build:                         *SUCCESS*
> Build: Completed at Thu Jul 13 17:24:11 UTC 2023
> Failed to create experiment:  *FAIL* C48_S2S
> Experiment setup: failed at Thu Jul 13 17:24:15 UTC 2023 for experiment C48_S2S
> 
> forecast-only mode treats ICs differently and cannot be staged here
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C48_S2S/EXPDIR/test/config.base as per user input.
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C48_S2S/EXPDIR/test/config.aeroanl as per user input.
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C48_S2S/EXPDIR/test/config.landanl as per user input.
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C48_S2S/EXPDIR/test/config.ocnanal as per user input.
> Failed to create experiment:  *FAIL* C96C48_hybatmDA
> Experiment setup: failed at Thu Jul 13 17:24:18 UTC 2023 for experiment C96C48_hybatmDA
> 
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96C48_hybatmDA/EXPDIR/test/config.base as per user input.
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96C48_hybatmDA/EXPDIR/test/config.aeroanl as per user input.
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96C48_hybatmDA/EXPDIR/test/config.landanl as per user input.
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96C48_hybatmDA/EXPDIR/test/config.ocnanal as per user input.
> Failed to create experiment:  *FAIL* C96_atm3DVar
> Experiment setup: failed at Thu Jul 13 17:24:21 UTC 2023 for experiment C96_atm3DVar
> 
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96_atm3DVar/EXPDIR/test/config.base as per user input.
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96_atm3DVar/EXPDIR/test/config.aeroanl as per user input.
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96_atm3DVar/EXPDIR/test/config.landanl as per user input.
> EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96_atm3DVar/EXPDIR/test/config.ocnanal as per user input.
> Created experiment:            *SUCCESS*
> Case setup: Completed at Thu Jul 13 18:04:22 UTC 2023 for experiment C48_S2S_4baaa621
> Created experiment:            *SUCCESS*
> Case setup: Completed at Thu Jul 13 18:04:26 UTC 2023 for experiment C96C48_hybatmDA_4baaa621
> Created experiment:            *SUCCESS*
> Case setup: Completed at Thu Jul 13 18:04:29 UTC 2023 for experiment C96_atm3DVar_4baaa621
> Experiment C48_S2S_4baaa621 completed: *SUCCESS*
> Experiment C48_S2S_4baaa621 Completed at Thu Jul 13 18:51:17 UTC 2023
> with 18 successfully completed jobs
> ```

@TerrenceMcGuinness-NOAA these logs are still not clearing properly when the process restarts.

**Comment by aerorahul** [comment]
> > Automated global-workflow Testing Results:
> > ```
> > Machine: Hera
> > Start: Thu Jul 13 15:56:54 UTC 2023 on hfe05
> > ---------------------------------------------------
> > Checkout:                      *SUCCESS*
> > Checkout: Completed at Thu Jul 13 16:01:41 UTC 2023
> > Build:                         *SUCCESS*
> > Build: Completed at Thu Jul 13 17:24:11 UTC 2023
> > Failed to create experiment:  *FAIL* C48_S2S
> > Experiment setup: failed at Thu Jul 13 17:24:15 UTC 2023 for experiment C48_S2S
> > 
> > forecast-only mode treats ICs differently and cannot be staged here
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C48_S2S/EXPDIR/test/config.base as per user input.
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C48_S2S/EXPDIR/test/config.aeroanl as per user input.
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C48_S2S/EXPDIR/test/config.landanl as per user input.
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C48_S2S/EXPDIR/test/config.ocnanal as per user input.
> > Failed to create experiment:  *FAIL* C96C48_hybatmDA
> > Experiment setup: failed at Thu Jul 13 17:24:18 UTC 2023 for experiment C96C48_hybatmDA
> > 
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96C48_hybatmDA/EXPDIR/test/config.base as per user input.
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96C48_hybatmDA/EXPDIR/test/config.aeroanl as per user input.
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96C48_hybatmDA/EXPDIR/test/config.landanl as per user input.
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96C48_hybatmDA/EXPDIR/test/config.ocnanal as per user input.
> > Failed to create experiment:  *FAIL* C96_atm3DVar
> > Experiment setup: failed at Thu Jul 13 17:24:21 UTC 2023 for experiment C96_atm3DVar
> > 
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96_atm3DVar/EXPDIR/test/config.base as per user input.
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96_atm3DVar/EXPDIR/test/config.aeroanl as per user input.
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96_atm3DVar/EXPDIR/test/config.landanl as per user input.
> > EDITED:  /scratch1/NCEPDEV/global/Terry.McGuinness/GFS_CI_ROOT/PR/1719/RUNTESTS/C96_atm3DVar/EXPDIR/test/config.ocnanal as per user input.
> > Created experiment:            *SUCCESS*
> > Case setup: Completed at Thu Jul 13 18:04:22 UTC 2023 for experiment C48_S2S_4baaa621
> > Created experiment:            *SUCCESS*
> > Case setup: Completed at Thu Jul 13 18:04:26 UTC 2023 for experiment C96C48_hybatmDA_4baaa621
> > Created experiment:            *SUCCESS*
> > Case setup: Completed at Thu Jul 13 18:04:29 UTC 2023 for experiment C96_atm3DVar_4baaa621
> > Experiment C48_S2S_4baaa621 completed: *SUCCESS*
> > Experiment C48_S2S_4baaa621 Completed at Thu Jul 13 18:51:17 UTC 2023
> > with 18 successfully completed jobs
> > ```
> 
> @TerrenceMcGuinness-NOAA these logs are still not clearing properly when the process restarts.

It might be a simple fix in that the `global-workflow.log` file is not being erased at the start of the (re-)run.

**Comment by TerrenceMcGuinness-NOAA** [comment]
This was an anomaly because I forced the restart on Hera without re-building on Hera today because it was only the case files in the PR that were wrong. Notice the Orion's started fresh.  I messed up the restart there and Walter added new case files to the PR and resubmitted it.

File: jobs/JGDAS_FIT2OBS (hunk 1 above)
Comment (WalterKolczynski-NOAA) [reply jobs/JGDAS_FIT2OBS]: Done

**Review by aerorahul — APPROVED** [review]
looks good.

**Pull request merged**